include/sound/soc-dapm.h

```diff
@@ -604,6 +604,7 @@ struct snd_soc_dapm_widget {
 	unsigned char power_checked:1;		/* power checked this run */
 	unsigned char is_supply:1;		/* Widget is a supply type widget */
 	unsigned char is_ep:2;			/* Widget is a endpoint type widget */
+	unsigned char no_wname_in_long_name:1;	/* No widget name prefix in kcontrol name */
```

Comment:
@jsarha you really need to give an example of 'before' and 'after', the commit message is really cryptic. I don't know how anyone could review this and give a thumbs up unless they wrote it themselves :-)

Reply:
This commit is a generic alsa-level commit, and I have no idea what kind of names the widgets may have, so its pretty hard to give a concrete example, but I try to rephrase the commit message.

Reply:
just provide the examples you are aware of?